src/snippets/figure.njk

```diff
@@ -0,0 +1,16 @@
+---
+title: Figure
+contributorsName: Michael W. Delaney
+contributorsURL: https://michaeldelaney.me
+templateEngineOverride: md, njk
+---
+{% set description %}
+  Usually a `<figure>` is an image, illustration, diagram, code snippet, etc. If your use case doesn't require a caption, the `<figcaption>` can be omitted.
```

Comment:
Maybe link to MDN docs on this.

I would also like to work in a summarized note from (with link to) [this article](https://thoughtbot.com/blog/alt-vs-figcaption) talking about the relationship between using figcaption _and_ img `alt`